piano.css

```diff
@@ -0,0 +1,75 @@
+html {
```

Comment:
linter error found, there must be 2 spaces between the beginning of the file and the code (horizontally).